worker/crlWorker/crlWorker.go

```diff
@@ -0,0 +1,117 @@
+package crlWorker
+
+import (
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sync"
+
+	"github.com/mozilla/tls-observatory/certificate"
+	"github.com/mozilla/tls-observatory/worker"
+)
+
+var (
+	workerName = "crlWorker"
+	workerDesc = "Checks certificate CRL (Certificate Revocation List) and reports on certificate revocation"
+)
+
+type Result struct {
+	Revoked bool `json:"revoked"`
+}
+
+type eval struct{}
+
+// Run implements the worker.Worker interface and is called check CRL status
+func (e eval) Run(in worker.Input, resChan chan worker.Result) {
+	result := worker.Result{
+		WorkerName: workerName,
+	}
+	crlRes := Result{}
+
+	// Grab first CRL response we can from our certificate
+	crlResp, err := e.getFirstCRLData(in.Certificate)
+	if err != nil {
+		result.Errors = append(result.Errors, err.Error())
```

Comment:
all of those errors should force the worker to exit immediately, something along the lines of 
```go
        if err != nil {
                res <- worker.Result{
                        Success:    false,
                        WorkerName: workerName,
                        Errors:     []string{fmt.Sprintf("failed to foo bar: %v", err)},
                        Result:     nil,
                }
        }
```